app/views/layouts/_aside.html.erb

```diff
@@ -0,0 +1,36 @@
+<table>
+<thead>
+</thead>
+<tbody>
+   <tr class="row">
+      <td class="h6 text-muted">
+         <hr>Find company
+      </td>
+   </tr>
+   <tr class="row">
+      <td>
+         <form class="d-flex">
+            <input class="form-control mr-sm-2" type="text" placeholder="Search" aria-label="Search">
+            <button class="btn btn-outline-secondary btn-sm" type="submit" role="button">Search</button>
+         </form>
+      </td>
+   </tr>
+   <tr>
+      <td>
+         <hr>
+         <a href="gpw/index.html" class="nav-link text-dark">&nbsp;GPW&nbsp;base</a>
```

Comment:
Why do you need these $nbzp;?

Reply:
is this link redirects user to a working page? if not maybe it is better to use href="#" now? Then it would "do nothing" which is probably better than displaying 404 not found error. But that is just a suggestion.

Reply:
is this link redirects user to a working page? if not maybe it is better to use href="#" now? Then it would "do nothing" which is probably better than displaying 404 not found error. But that is just a suggestion.

Reply:
The begin &nbsp; of course don't need, but the middle I like to keep to see links in one line in smaller windows.
Links will change for now href='#' :)